src/test/external-test/ext_out_cluster_init_test.go

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package external_test provides a test for the external init flow.
+package external_test
+
+import (
+	"path"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/pkg/utils/exec"
+	test "github.com/defenseunicorns/zarf/src/test/e2e"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExtOutClusterDeploy(t *testing.T) {
+	// Docker/k3d networking constants
+	network := "k3d-k3s-external-test"
+	subnet := "172.31.0.0/16"
+	gateway := "172.31.0.1"
+	giteaIP := "172.31.0.99"
+	giteaHost := "gitea.localhost"
+	registryHost := "registry.localhost"
+
+	zarfBinPath := path.Join("../../../build", test.GetCLIName())
+
+	// Teardown any leftovers from previous tests
+	_ = exec.CmdWithPrint("k3d", "cluster", "delete")
+	_ = exec.CmdWithPrint("k3d", "registry", "delete", registryHost)
+	_ = exec.CmdWithPrint("docker", "network", "remove", network)
```

Comment:
this would probably make a local dev real sad if their network were killed, should we put some sort of warning on that?

Reply:
It would only kill the named network and registry used by the test.  It would kill their k3d cluster though if they were using the default name.  We could name that too?

Reply:
ah n/m I see this is removing only that k3d network, disregard.